modules/components/src/Admin/Dashboard.js

```diff
@@ -519,7 +524,7 @@ class Dashboard extends React.Component {
           />
           <Route
             exact
-            path="/projects/:projectId/:index"
+            path={basePath + '/:projectId/:index'}
```

Comment:
Can you avoid concating basePath and passing it around by making a router for the dashboard and setting `basename` on it?

Reply:
I didn't think you could nest routers. I'll try it

Reply:
this does seem to work. is this what you mean?

```
  render() {
    return (
      <BrowserRouter>
      <div>
        <Link to="/admin">go</Link>
        <Route path="/admin" render={({ match }) =>
          <BrowserRouter basename={match.url}>
            <div>
              <Route path="/" render={() => <div>sup</div>} />
            </div>
          </BrowserRouter>
        } />
      </div>
      </BrowserRouter>
    );
  }
```

Reply:
this seems far better than what the official docs suggest for nested routing

Reply:
yeah thats exactly what I was thinking :) it works? yay! 
whats the official method?

Reply:
they just don't go that far to say nest routers. they're probably being conservative considering you rely on nesting context 

https://reacttraining.com/react-router/core/guides/philosophy/nested-routes